modules/persistent-storage-csi-efs-sts.adoc

```diff
@@ -0,0 +1,94 @@
+
+
+.Prerequisites
+
+* AWS account credentials
+
+.Procedure
+
+To configure AWS EFS CSI Driver Operator with STS:
+
+. Get the ccoctl tool from {product-title} image, which you used to install the cluster with STS (see xref:../../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#cco-ccoctl-configuring_cco-mode-sts[Configuring the Cloud Credential Operator utility]).
+
+. Copy the EFS CredentialsRequest (https://github.com/openshift/aws-efs-csi-driver-operator/blob/master/assets/credentials.yaml), save it, and then place it in the `credrequests` directory.
++
+.Example
++
+[source, yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: openshift-aws-efs-csi-driver
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AWSProviderSpec
+    statementEntries:
+    - action:
+      - elasticfilesystem:*
+      effect: Allow
+      resource: '*'
+  secretRef:
+    name: aws-efs-cloud-credentials
+    namespace: openshift-cluster-csi-drivers
+  serviceAccountNames:
+  - aws-efs-csi-driver-operator
+  - aws-efs-csi-driver-controller-sa
+----
+
+.  Use ccoctl to generate a new IAM role in AWS, and create a yaml file for it in the local filesystem (`manifests/openshift-cluster-csi-drivers-aws-efs-cloud-credentials-credentials.yaml`).
++
+[source, terminal]
+----
+$ ccoctl aws create-iam-roles --name=<name> --region=<aws_region> --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests --identity-provider-arn=arn:aws:iam::<aws_account_id>:oidc-provider/<name>-oidc.s3.<aws_region>.amazonaws.com
+----
++
+.Example
++
+[source, terminal]
+----
+$ ./ccoctl aws create-iam-roles --name wduan-0321a-aw-9163 --credentials-requests-dir /home/wduan/01Work/testFile/aws-efs/sts --identity-provider-arn arn:aws:iam::301721915996:oidc-provider/wduan-0321a-aw-9163-oidc.s3.us-east-2.amazonaws.com
+----
++
+.Output
++
+[source, terminal]
+----
+2022/03/21 06:24:44 Role arn:aws:iam::301721915996:role/wduan-0321a-aw-9163-openshift-cluster-csi-drivers-aws-efs-cloud- created
+2022/03/21 06:24:44 Saved credentials configuration to: /home/wduan/01Work/config/manifests/openshift-cluster-csi-drivers-aws-efs-cloud-credentials-credentials.yaml
+2022/03/21 06:24:45 Updated Role policy for Role wduan-0321a-aw-9163-openshift-cluster-csi-drivers-aws-efs-cloud-
+----
+
+. Create the AWS EFS cloud credentials and secret:
++
+[source, terminal]
+----
+$ oc create -f <path/to/secret file>
```

Comment:
In the previous step, we mentioned "Use ccoctl to generate a new IAM role in AWS, and create a YAML file for it in the local filesystem", we could use the <path_to_ccoctl_output_dir>/manifests/openshift-cluster-csi-drivers-aws-efs-cloud-credentials-credentials.yaml here, it will be more clear.